stm32-modules/flex-stacker/firmware/motor_control/motor_hardware.c

```diff
@@ -285,7 +287,7 @@ void hw_enable_motor(MotorID motor_id) {
             port = Z_EN_PORT;
             pin = Z_EN_PIN;
             status = HAL_TIM_Base_Start_IT(&htim20);
-            HAL_NVIC_SetPriority(TIM20_UP_IRQn, 5, 0);
+            HAL_NVIC_SetPriority(TIM20_UP_IRQn, 10, 0);
             HAL_NVIC_EnableIRQ(TIM20_UP_IRQn);
             if (status == HAL_OK)
```

Comment:
can we remove these if we are not using them, we are also returning a bool based on the status so I don't think we need this.

Reply:
Good call! I actually updated these in the next PR, so i'll leave these as such right now